### DISCUSSION TITLE
Order by [Uptime] , not working at all

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#96] [General]
just see below : 

https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/71503e7d-3562-4934-9035-8ff94632d852

**Comment by TJ-BitCanna** [comment]
We'll look into it :)

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- debug voting on uptime on the Validators page

**Comment by TJ-BitCanna** [comment]
#115

**Comment by TJ-BitCanna** [comment]
Completed :)
